src/main/java/com/solvd/laba/MyConnectionPoolNew1.java

```diff
@@ -0,0 +1,67 @@
+package com.solvd.laba;
+
+import java.util.concurrent.*;
+
+public class MyConnectionPoolNew1 {

```

Comment:
Why do you have MyConnectionPool and MyConnectionPoolNew1 ? 
This does not look like a production ready code. Clean it and send for re-review.

Reply:
@VbKoshak - I have updated my connection pool project. Please re-review and let me know your thoughts. Thanks!